src/services/git/GitHubService.ts

```diff
@@ -136,7 +136,7 @@ export class GitHubService implements IVCSService {
       closedAt: response.data.closed_at,
       mergedAt: response.data.merged_at,
       state: response.data.state,
-      id: response.data.id,
+      id: response.data.number,
```

Comment:
How do you know that `number` attribute is an PR id?

- [ ] Add a link referring to the information why the `number` is the correct PR id.

I have found just this:
> Be aware that the id of a pull request returned from "Issues" endpoints will be an issue id. To find out the pull request id, use the "List pull requests" endpoint.

src.: https://developer.github.com/v3/issues/#list-issues
... but nothing about the `number` attribute.